target/linux/realtek/image/rtl838x.mk

```diff
@@ -36,6 +36,16 @@ define Device/d-link_dgs-1210-28
 endef
 TARGET_DEVICES += d-link_dgs-1210-28
 
+define Device/engenius_ews2910p
+  SOC := rtl8380
+  IMAGE_SIZE := 8192k
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := EWP2910P
+  UIMAGE_MAGIC := 0x03802910
+  UIMAGE_NAME := IMG-2.00.42-c2.0.69
```

Comment:
Is this an existing firmware number, of does it just have to match a certain format?

Reply:
Can you use `IMG-0.00.00-c0.0.00` as fake version number? With the value you suggested, it appears like a minimal version number is required. If you the firmware only nags about the controller version, that's not an issue (which is also what you have now).

Reply:
Sure, I can test this.

Reply:
Seems to work
![image](https://user-images.githubusercontent.com/2610559/175185726-faf8e71f-b2ae-410f-a772-4ee8a05215b4.png)